src/components/Profile/components/PersonalInfoList/PersonalInfoList.module.scss

```diff
@@ -24,34 +24,19 @@
   }
 }
 
-.my-personal-info {
-  & .private {
-    color: $neutral-black-opacity40;
-
-    & .not-private {
-      opacity: 1;
-      color: $dark-text-color;
-    }
-  }
+// Needed for mail implementation, should be refactored into PersonalInfoListItem
```

Comment:
Getting these two components to display which info is or is not private is SURPRISINGLY complex.